src/reports/package/footer-text-for-service.tsx

```diff
@@ -2,18 +2,28 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
 import { ReportSectionFactory } from 'reports/components/report-sections/report-section-factory';
 import { ToolLink } from 'reports/components/report-sections/tool-link';
+import { NamedFC } from 'common/react/named-fc';
 
+export type FooterTextProps = { environmentInfoProvider: EnvironmentInfoProvider };
 export const FooterTextForService = (serviceName: string) => {
 
-    const footerText = ({ environmentInfo: { axeCoreVersion, browserSpec } }) =>
-        <>
-            This automated checks result was generated using the {serviceName}{' '}
-            that helps find some of the most common accessibility issues. The scan was
-            performed using axe-core {axeCoreVersion} and {browserSpec}. For a complete
-            WCAG 2.1 compliance assessment please visit{' '}<ToolLink />.
-        </>;
+    const footerText = NamedFC<FooterTextProps>(
+        'FooterText',
```

Comment:
This component has the same 'displayName` as the original 'FooterText'. One of the things this 'displayName' is useful for is debugging, where having different names can help you identify the actual component being use to render some part of the dom. Not terribly important, but something to consider.

Reply:
I don't see any changes on the `displayName` since I left the original comment (that would be `'FooterText'`). I'm not sure   it's a good idea to keep both components using the same `displayName` and considering the change is easy enough, I rather have it updated.